tck/src/test/java/io/smallrye/concurrency/tck/SmallRyeConcurrencyArchiveProcessor.java

```diff
@@ -50,6 +52,8 @@ public void process(Archive<?> applicationArchive, TestClass testClass) {
                 archive.addAsLibrary(newJar);
             }
             archive.addAsResource("jndi.properties");
+            archive.addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
```

Comment:
Quite surprised this won't interfere with TCKs that already add beans.xml. I suppose shrinkwrap somehow handles it already.

Reply:
Well I need someone to provide a beans.xml. I did run the full test suite with this PR and all tests pass.